shared-bindings/socketpool/SocketPool.c

```diff
@@ -58,7 +58,7 @@ STATIC mp_obj_t socketpool_socketpool_make_new(const mp_obj_type_t *type, size_t
 }
 
 
-//|     def socket(self, family: int = AF_INET, type: int = SOCK_STREAM, proto: int = IPPROTO_TCP) -> None:
```

Comment:
Is there no way for this to include these defaults? They are helpful to have in docs.

Reply:
I went back and forth on this - they aren't currently defined in the python class.  I made it match the socket constructor:
```
//| class socket:
//|
//|     def __init__(self, family: int, type: int, proto: int) -> None:
//|         """Create a new socket
```
I'll update both of these to define them in the python class

Reply:
Take a look now and let me know what you think - I added class attributes for the enum/define values

Reply:
Is it deliberate that AF_INET is defined twice with different numeric values?

Reply:
That is a good question - I matched what's in the C code, so the bigger question is does it make sense for the C header values to be the same?

`network/__init__.h`
```C
#define MOD_NETWORK_AF_INET (2)
#define MOD_NETWORK_AF_INET6 (10)

#define MOD_NETWORK_SOCK_STREAM (1)
#define MOD_NETWORK_SOCK_DGRAM (2)
#define MOD_NETWORK_SOCK_RAW (3)
```

`socketpool/SocketPool.h`
``` C
typedef enum {
    SOCKETPOOL_SOCK_STREAM,
    SOCKETPOOL_SOCK_DGRAM,
    SOCKETPOOL_SOCK_RAW
} socketpool_socketpool_sock_t;

typedef enum {
    SOCKETPOOL_AF_INET,
    SOCKETPOOL_AF_INET6
} socketpool_socketpool_addressfamily_t;
```

Reply:
Can we define them as `AF_INET = ...`? That way the implementation of the number stays an implementation detail.

Reply:
Removed the implementation-specific values from the stubs